submissions/alliEse/HTML/index.html

```diff
@@ -6,7 +6,14 @@
     <title>Essential Fantasies - Wingtip Brogues</title>
   </head>
   <body>
-    <nav><h1>Essential Fantasies</h1></nav>
+    <nav><h1>Essential Fantasies</h1>
+    <ul>
+				<li><a href="#"> Home</a></li>
+				<li><a href="#">Blog</a> </li>
+				<li><a href="index.html">Productpage</a> </li>
+				<li><a href="../html-forms/contact.html">Contact Us</a> </li>
```

Comment:
Well done @EseAlli 

Linking of pages will be done in your next task. You can remove them for now